feed_checker.py

```diff
@@ -10,29 +11,18 @@
 
 
 def clean_url(url):
+    if not url:
+        raise Exception()
     url = urllib.parse.urlparse(url)
     query = urllib.parse.parse_qs(url.query, keep_blank_values=True)
     query.pop("api_key", None)
     query.pop("token", None)
```

Comment:
@evansiroky Here's the part where it removes sensitive parameters. It's not in green because I unplugged this when I removed the "organize results by domain" logic (all the places with `clean_url` below are using this. 

Are there any other keys we should be removing? This would be pretty easy to cover with unit tests to avoid similar accidents in the future.

Reply:
Thanks for highlighting this for me.